### DISCUSSION TITLE
Formula using IF [Oct 08]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6695] [Discord Threads]
> 🙋‍♂️ **soggydoughnuts**

**Comment by o1lab** [comment]
> 👨‍🦲 **dstala**  
#### Something like this works   
#### IF({Country}=='United States','TRUE','FALSE')  
#  
> 👨‍🦲 **dstala**  
#### You can find more options here   
#### https://docs.nocodb.com/setup-and-usages/formulas   
####    
#### Do let us know your feedback on what you find it hard to figure it out. We are revising our documentation- will incorporate  
#  
> 🙋‍♂️ **soggydoughnuts**  
#### Ok great!  I will take a look.    
#### I think just some of the formula usage being different is an adjustment coming from airtable.    
####    
#### Example: I had to swap SUBSTITUE to REPLACE and then my formula worked fine  
#  
> 🙋‍♂️ **soggydoughnuts**  
#### It seems that the `&` doesn't work?  
#  
> 👨‍🦲 **dstala**  
#### You can share the formula functionality that didn't work; we will check it out. Help us by creating an issue with associated details in github (or share details here- whatever works for you)  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1149191123634491422)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
